### DISCUSSION TITLE
[Announment] Welcome new PMC Member: Joe Zou

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3762] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that [Joe Zou](http://github.com/zouyx) a.k.a zouyx, has been voted in as a new Apollo PMC Member.

Please join me to say congratulations to him!

@zouyx, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by kezhenxu94** [comment]
Congratulations and welcome!! @zouyx

**Comment by zouyx** [comment]
Thanks @nobodyiam @kezhenxu94 . I am grad to join Apollo PMC.

**Reply by darren-fu** [reply]
niubility
